### ISSUE TITLE
Filtering by mongo id do not work properly. 

### ISSUE THREAD

**Issue by piron4o** [sonata-project/SonataDoctrineMongoDBAdminBundle#131]
I think there must be new filter type for mongo id because type "doctrine_mongo_string" doesn't work for it.

``` php
 $queryBuilder->field($field)->equals(new \MongoRegex(sprintf('/%s/i', $data['value'])));
```

**Comment by joshlopes** [comment]
This is fixed on the PR #141 -- I am checking the type of it and converting to mongoID ;)

**Comment by jordisala1991** [comment]
Probably #141 could  be rescued
